### PR TITLE
Initial draft or persistent volume recycler deprecation statement.

### PR DESCRIPTION
**Pull request by childsb**
As discussed in the storage SIG this is a draft Notice of Deprecation for the persistent volume recycler API.

### CODE REVIEW

**Comment by rootfs** [comment]
should this proposal live inside `pkg/controller/persistentvolume/` or `docs/proposals`?

File: pkg/controller/persistentvolume/deprecated-recycle-api.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Deprecation of Persistent Volume Recycler Controller behavior
+The function of the persistent volume recycler will change in future kubernetes releases.  The current implementation will remain and function as is for the short term but expected to be removed or replaced in future releases.  
+
+## Summary
+The current recycler design only allows for user space volume erasure operations.  This is inadequate for control plane or scrubbing that requires elevated privileges.  Future redesign is anticipated and this document serves as warning of future API change. **USE RECYCLE API AT YOUR OWN RISK AS IT MAY BE REMOVED OR SIGNIFICANTLY CHANGED IN FUTURE KUBERNETES RELEASES**.
+
```
Comment (wattsteve) [inline pkg/controller/persistentvolume/deprecated-recycle-api.md]: Can you be a little more explicit about what the limitations are with the current recycler? Perhaps you could walk through a scenario which demonstrates its limitations? Also, why do you we need control plane or elevated privileges? Perhaps you could cover that in your scenario as well.

**Comment by eparis** [comment]
@k8s-bot unit test this

**Comment by thockin** [comment]
I don't think we're proceeding with this, are we?

**Pull request closed without merging**